content/blog/2025-02-17-datafusion-comet-0.6.0.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: Apache DataFusion Comet 0.6.0 Release
+date: 2025-02-17
+author: pmc
+categories: [subprojects]
+---
+
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache DataFusion PMC is pleased to announce version 0.6.0 of the [Comet](https://datafusion.apache.org/comet/) subproject.
+
+Comet is an accelerator for Apache Spark that translates Spark physical plans to DataFusion physical plans for
+improved performance and efficiency without requiring any code changes.
+
+Comet runs on commodity hardware and aims to provide 100% compatibility with Apache Spark. Any operators or
+expressions that are not fully compatible will fall back to Spark unless explicitly enabled by the user. Refer
+to the [compatibility guide] for more information.
+
+[compatibility guide]: https://datafusion.apache.org/comet/user-guide/compatibility.html
+
+This release covers approximately four weeks of development work and is the result of merging 39 PRs from 12
+contributors. See the [change log] for more information.
+
+Starting with this release, we now plan on releasing new versions of Comet more frequently, typically within 1-2 weeks
+of each major DataFusion release. The main motivation for this change is to better support downstream Rust projects 
+that depend on the [datafusion_comet_spark_expr] crate.
+
+[datafusion_comet_spark_expr]: https://docs.rs/datafusion-comet-spark-expr/latest/datafusion_comet_spark_expr/
+
+[change log]: https://github.com/apache/datafusion-comet/blob/main/dev/changelog/0.6.0.md
+
+## Release Highlights
+
+### DataFusion Upgrade
+
+- Comet 0.6.0 uses DataFusion 45.0.0
+
+### New Features
+
+- Comet now supports `array_join`, `array_intersect`, and `arrays_overlap`.
+
+### Performance & Stability
+
+- Metrics from native execution are now updated in Spark every 3 seconds by default, rather than for each
```

Comment:
is it async update? if so its probably worth mentioning?

Reply:
No, it is still synchronous